frontend/my-app/src/components/GenMeetingBarChart.js

```diff
@@ -0,0 +1,79 @@
+import React, {useState, useEffect} from 'react'
+import {Chart as ChartJS, BarElement, CategoryScale, LinearScale} from 'chart.js'
+import {Bar} from 'react-chartjs-2'
+
+ChartJS.register(
+    CategoryScale,
+    LinearScale,
+    BarElement
+)
+
+
+const GenMeetingBarChart = () => {
+
+    const [chart, setChart] = useState([]);
```

Comment:
Maybe for later iterations but when you're setting state that will take time to render because of backend, its good to have an isLoading variable that stalls loading the state dependent UI before that state is fully processed